cli/js/tests/test_util.ts

```diff
@@ -235,7 +235,14 @@ export function createResolvable<T>(): Resolvable<T> {
   return Object.assign(promise, methods!) as Resolvable<T>;
 }
 
-export class SocketReporter implements Deno.TestReporter {
+export enum TestEvent {
```

Comment:
I think previous solution was cleaner, don't like that it has to redeclared and kept in sync

Reply:
That's the thing, they don't have to be kept in sync. The mechanisms are completely independent even though they work similarly. This is for the sake of keeping `kind` out of the custom `Reporter` API where they are redundant (because each message type gets passed to a different callback).